vtex/faststore
# Feat: Add and update `Newsletter` component

## What's the purpose of this pull request?
`fs/components`
- [x] Moves `Newsletter` component

`fs/ui`
- [x] Updates `Newsletter` styles
- [ ] Adds new `colorVariant` variants 
 
`fs/core`
- [x] Review `Newsletter ` structure @lucasfp13 

## How to test it?

run `yarn dev` and check the `Newsletter` feature on the store or test it using the doc preview link:
- [Newsletter Doc](https://faststore-site-git-feat-newsletter-fs-247-faststore.vercel.app/components/organisms/newsletter)

### Starters Deploy Preview

`WIP`
